api/cas-server-core-api-configuration/src/main/java/org/apereo/cas/configuration/support/CasConfigurationJasyptCipherExecutor.java

```diff
@@ -163,14 +165,24 @@ public String getName() {
      */
     public String encryptValue(final String value) {
         try {
-            initializeJasyptInstanceIfNecessary();
-            return ENCRYPTED_VALUE_PREFIX + this.jasyptInstance.encrypt(value);
+            return encryptValue2(value);
         } catch (final Exception e) {
             LOGGER.error("Could not encrypt value [{}]", e);
         }
         return null;
     }
 
+    /**
+     * Encrypt value string (but don't log error, for use in shell).
+     *
+     * @param value the value
+     * @return the string
+     */
+    public String encryptValue2(final String value) {
```

Comment:
Please rename these methods, removing the number 2, into something that is a bit more intention-revealing.